recipes/rotor/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class RotorConan(ConanFile):
+    name = "rotor"
+    license = "MIT"
+    homepage = "https://github.com/basiliscos/cpp-rotor"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = (
+        "Event loop friendly C++ actor micro-framework, supervisable"
+    )
+    topics = ("concurrency", "actor-framework", "actors", "actor-model", "erlang", "supervising", "supervisor")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
```

Comment:
No shared library support?

Reply:
Yes, I plan to do that in the next rotor release, in the current one - no, only static.

Reply:
what about adding it now, then raising a ConanInvalidConfiguration? It will be better to the Package ID, as always have shared option available.

Reply:
Good idea, thank you. I'll add that in conan recipe !